src/aiidalab_qe/common/process.py

```diff
@@ -55,13 +55,29 @@ def __init__(self, process_label, **kwargs):
         else:
             self.fmt_workchain = self.BASE_FMT_WORKCHAIN
 
-        self.refresh_work_chains_button = ipw.Button(description="Refresh")
+        self.new_work_chains_button = ipw.Button(
+            description="New",
+            tooltip="Start a new computation",
+            button_style="primary",
```

Comment:
I would suggest using different colors for the buttons `New` and `Refresh`.
```suggestion
            button_style="success",
```